lib/widgets/transaction_details.dart

```diff
@@ -78,17 +79,25 @@ class TransactionDetails extends StatelessWidget {
       ThemeData theme, ValueTransferOutput output, bool isLastOutput) {
     final extendedTheme = theme.extension<ExtendedTheme>()!;
     Widget timelock = SizedBox(height: 0);
-    //TODO(#505): Support timelocks
-    // if (output.timeLock != 0) {
-    //   timelock = Expanded(
-    //       child: Column(
-    //           crossAxisAlignment: CrossAxisAlignment.end,
-    //           mainAxisAlignment: MainAxisAlignment.end,
-    //           children: [
-    //         Text(output.timeLock.toInt().formatDate(),
-    //             style: theme.textTheme.bodySmall)
-    //       ]));
-    // }
+    if (output.timeLock != 0) {
+      timelock = Row(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          mainAxisAlignment: MainAxisAlignment.start,
+          children: [
+            Icon(DateTime.now().millisecondsSinceEpoch ~/ 1000 <
+                    output.timeLock.toInt()
+                ? FontAwesomeIcons.lock
```

Comment:
Icon should be smaller and align with the date
<img width="385" alt="Screenshot 2024-04-04 at 16 42 59" src="https://github.com/witnet/my-wit-wallet/assets/16032491/9c55e2af-b188-426b-a7ec-1aefd5651b7c">